README.md

```diff
@@ -1,7 +1,7 @@
 # Introduction
 
-`eslint-plugin-svelte` is [ESLint] plugin for [Svelte].  
```

Comment:
The two spaces at the end of the line are intended to be `<br>`. Please undo the change or insert `\` instead.

Reply:
I recently learned that I can use backslashes in markdown.
It would be nice for me if a formatter or something could fix trailing spaces to backslashes, but I haven't found it yet 😓

Reply:
You could try `sed -i "s/  $/ \\/"` maybe? You might just need to review the changes in case changes anything unexpected.